### DISCUSSION TITLE
bandpass filters

### DISCUSSION THREAD

**Discussion by nipper01792** [pschatzmann/arduino-audio-tools#365] [Q&A] [answered]
how would I go about creating a bandpass filter passing between 100 and 15k

**Comment by pschatzmann** [comment, marked as answer]
Did you read the documentation: https://github.com/pschatzmann/arduino-audio-tools/wiki/Filters ?
It includes a link to a filter design tool...

**Comment by nipper01792** [comment]
I have now  re read and found the link. I must have missed it last time . That will teach me for skimming over stuff. Thank you
